src/modules/hip/kernel/bitwise_and.hpp

```diff
@@ -0,0 +1,247 @@
+#include <hip/hip_runtime.h>
+#include "rpp_hip_common.hpp"
+
+/* BitwiseAND is logical operation only on U8/I8 types.
```

Comment:
We need to create a "logical_operations" header for bitwise ops. Currently everything is under arithmetic.
Test suite grouping/classification, external .h include, and internal .hpp includes need to change.

Reply:
Done, please recheck.
Confirmed QA test pass too.